inbox/automatic-trust-transfer.xml

```diff
@@ -0,0 +1,340 @@
+		<remark><p>First draft.</p></remark>
+	</revision>
+</header>
+<section1 topic='Introduction' anchor='intro'>
+	<p>
+		ATT is used for automatically establishing secure channels protected against active attacks between a new device and existing ones after a single mutual manual authentication between the new device and one of the existing ones.
+		It preserves the security level as if all devices had authenticated their keys manually.
+		A trusted third party is not required since a usual OMEMO message is used for transferring the information needed to authenticate a key or revoke the trust in that key.
+		Additionally, it will preserve the anonymity of the authentication and revocation since those messages are only sent to devices with authenticated keys.
+		That means that an attacker cannot detect whether an authentication or revocation took place.
+	</p>
+	<p>
+		End-to-end encryption without verifying the authenticity of the public keys enables users to protect their communication against passive attacks.
+		That means an attacker cannot read the transferred messages without manipulating the exchanged messages or key material.
+		But without any other precautions active attacks are still possible.
+		If the exchanged keys are replaced with the key of an attacker, the end-to-end encrypted messages can be read by the attacker.
+	</p>
+	<p>
+		When using &xep0384;, a public identity key is transmitted over a channel which is not protected against active attacks.
+		That key has to be authenticated by the receiving device over a channel which is protected against active attacks to maintain the confidentiality of sent OMEMO messages and ensuring the authenticity and integrity of received OMEMO messages.
+	</p>
+	<p>
+		When using OMEMO, each device has a different identity key.
+		That makes it possible for new devices to use end-to-end encryption protecting against passive attacks without transmitting the private key over a secure channel from an existing device to the new one.
+		However, the downside of this approach is that it increases the number of authentications.
+	</p>
+	<p>
+		The goal of key authentication is to build up an end-to-end encrypted communication network exclusively between devices with authenticated keys.
+		That network of devices trusting each other's keys can be seen as a complete graph with each device as a node and each authentication as an edge.
+		The number of edges grows for each new device by the number of existing nodes.
+		Without ATT all of those authentications have to be done manually.
+		With ATT though, only one mutal manual authentication is required.
+	</p>
+	<p>
+		This means that each communication channel between the devices is resistant against active attacks.
+		To sustain such a secure communication across all devices, the new key of an own device has to be authenticated by all n own devices and all m devices of a contact.
+		This leads to a total of n * m authentications.
```

Comment:
@melvo Isn't this `n + m` authentications when adding a new key?

Reply:
Hi @jcbrand, thanks for your review :)
Yes. Of course, you are right. I am working on a revised version that clarifies it.